frontend/src/pages/LandingPage.jsx

```diff
@@ -22,14 +26,65 @@ const LandingPage = () => {
             Experience money transfer in a whole new way with P2Max
           </p>
           <div className="mt-6 pb-6 text-center md:text-left md:pb-0">
-            <button className="flex justify-center items-center gap-4" id="btn">  <img src={hero_btn_icon} alt="hero_btn_icon" />SEND MONEY NOW</button>
+            <button className="flex justify-center items-center gap-4" id="btn">
```

Comment:
It would be great if we have a button component that is reusable.

Reply:
yeah we would, this needs to be merged and pulled into everyone's local first

Reply:
Why how?

If the button were a reusable component, the PR could still be merged and everyone can pull to their local